src/realm.h

```diff
@@ -3088,6 +3102,43 @@ RLM_API void realm_sync_config_set_recovery_directory_path(realm_sync_config_t*,
 RLM_API void realm_sync_config_set_resync_mode(realm_sync_config_t*,
                                                realm_sync_session_resync_mode_e) RLM_API_NOEXCEPT;
 
+// flx sync c-api
+RLM_API realm_flx_sync_subscription_set_t* realm_sync_get_latest_subscription_set(const realm_t* realm);
+RLM_API realm_flx_sync_subscription_set_t* realm_sync_get_active_subscription_set(const realm_t* realm);
+RLM_API realm_flx_sync_mutable_subscription_set_t*
+realm_sync_get_mutable_subscription_set(const realm_flx_sync_subscription_set_t* subscription_set);
+RLM_API void realm_sync_get_state_change_notification_async(const realm_flx_sync_subscription_set_t* subscription_set,
+                                                            realm_flx_sync_subscription_set_state_e notify_when,
+                                                            realm_sync_on_subscription_state_changed callback);
+RLM_API int64_t realm_sync_subscription_version(const realm_flx_sync_subscription_set_t* subscription_set)
+    RLM_API_NOEXCEPT;
+RLM_API realm_flx_sync_subscription_set_state_e
+realm_sync_subscription_state(const realm_flx_sync_subscription_set_t* subscription_set) RLM_API_NOEXCEPT;
+RLM_API const char*
+realm_sync_subscription_error_str(const realm_flx_sync_subscription_set_t* subscription_set) RLM_API_NOEXCEPT;
+RLM_API size_t realm_sync_subscription_set_size(const realm_flx_sync_subscription_set_t* subscription_set)
+    RLM_API_NOEXCEPT;
+RLM_API realm_flx_sync_subscription_t* realm_sync_subscription_at(realm_flx_sync_subscription_set_t* subscription_set,
+                                                                  size_t index);
+RLM_API realm_flx_sync_subscription_json_t*
+realm_sync_subscription_to_ext_json(const realm_flx_sync_subscription_set_t* subscription_set);
```

Comment:
I don't think we need to expose this to the C-API for now. It's used internally by the sync client only.

Reply:
OK perfect, because it required a custom type to be returned.